infrastructure/tfvars_tests/standard-gke-public.platform.tfvars

```diff
@@ -49,3 +49,36 @@ cpu_pools = [{
   disk_size_gb   = 100
   disk_type      = "pd-standard"
 }]
+
+## make sure required gpu quotas are available in the corresponding region
+enable_gpu = true
```

Comment:
Our tests may be very flaky if we rely on GPUs on every run. Any way we can run the test with CPUs only?

Reply:
We discussed only triggering this once per PR via a manual job trigger by the author (by commenting "/run e2e"). Hopefully that won't fail due to stockouts. I prefer to not test with CPUs since a lot of the functionality relies on the application running on GPUs (like ray, mistral) - so if someone messed up the node selectors, we wouldn't catch it.

Reply:
Okay fair, but the test right now is not actually testing anything that needs GPUs right? We're just checking if Ray is up and the RAG frontend is running. So we don't need to add GPUs yet

Reply:
The check is now triggered only when a comment /gcbrun is added.
@andrewsykim Yes, in this PR we are only checking if frontend is running. We'll be performing the test while checking the response from the inference server in another PR. GPUs would be required then.